### ISSUE TITLE
Associating an IP with Tags to a Network.

### ISSUE THREAD

**Issue by rvalle** [ngine-io/ansible-collection-cloudstack#47]
I am having trouble with cs_ip_address and custom ID

```
- name: Associate an IP address with tags
  ngine_io.cloudstack.cs_ip_address:
    network: My Network
    tags:
      - key: myCustomID
      - value: 5510c31a-416e-11e8-9013-02000a6b00bf
  register: ip_address
```

I was under the assumption that the previous statement would do 2 things: assign an IP to My Network and tag it for future reference. Is that assumption correct?

I am not managing to use this module to assign a random public IP to a network in an idempotent way.

**Comment by rvalle** [comment]
arrrrghhhh, the problem is that I wrongly copied YAML from the sample in the documentation.

both Key and Value have a dash with is wrong.

It must be like this:

```
- name: Associate an IP address with tags
  ngine_io.cloudstack.cs_ip_address:
    network: My Network
    tags:
      - key: myCustomID
        value: 5510c31a-416e-11e8-9013-02000a6b00bf
  register: ip_address
  ```

spot the difference?

**Comment by rvalle** [comment]
the minor doc typo was fixed in #48
